community/extra-cmake-modules/version

```diff
@@ -1 +1 @@
-5.85.0 1
+5.86.0 2
```

Comment:
The epoch is always reset on maj/min/patch version bumps, so the version string should simply be `5.86.0 1` (epoch only increments on substantial build changes).

Reply:
I see! I'll go ahead and fix that.

Reply:
..Er, wait, when I do, what should the commit message be? And if I make a meta change wouldn't it go back to being "2"? This is a mess >_<

Reply:
If you force push the commit (I don't do this often so this is a guess) it should just push as being part of the most recent commit, so it will simply ammend these content changes (not the commit message).

Reply:
Should be fixed now.